config/standard/test/targets.spec.js

```diff
@@ -3,7 +3,7 @@
 // chai.use(chaiExclude);
 // const assert = chai.assert;
 const assert = require('chai').assert;
-const NootilsManager = require('medic-nootils/src/node/test-wrapper');
+const NootilsManager = require('@kennsippell/medic-nootils/src/node/test-wrapper');
```

Comment:
Is this meant to be linking to the medic version instead?

Reply:
Yep - this is normal for testing a package when the change has yet to pass code review. Diana approved the change this morning, so I will now remove it and publish the live `medic-conf 3.0.0` package